src/cli/bootstrap/bootstrap.ts

```diff
@@ -165,17 +170,20 @@ async function createTokenDistribution() {
     console.log('Create token distribution');
     const totalSupply = 50000000000.0;
     const allocations = {
-        'ecosystm.tmy': 0.3 * totalSupply,
-        'team.tmy': 0.15 * totalSupply,
-        'coinsale.tmy': 0.15 * totalSupply,
-        'ops.tmy': 0.4 * totalSupply,
+        'ecosystm.tmy': { id: 1, percentage: 0.3, amount: 0.3 * totalSupply },
+        'team.tmy': { id: 2, percentage: 0.15, amount: 0.15 * totalSupply },
+        'coinsale.tmy': { id: 3, percentage: 0.15, amount: 0.15 * totalSupply },
```

Comment:
coinsale.tmy does not get allocated to the vesting contract. it needs to be transferred to the coinsale.tmy account, we will then use that account to allocate tokens to the user in the vesting contract